docs/guides/getting-started/index.html

```diff
@@ -0,0 +1,81 @@
+---
+title: Getting Started
```

Comment:
Add what they are getting started with..."Getting started with xxx". Also, just initial capitalization is fine, not necessary on all words unless they are proper nouns.

Reply:
Title Case dictates that `Started` should be capitalized, correct?